packages/sw-routing/src/lib/router.js

```diff
@@ -52,11 +52,24 @@ class Router {
   /**
    * Start with an empty array of routes, and set up the fetch handler.
    */
-  constructor() {
+  constructor({handleFetch} = {}) {
+    if (typeof handleFetch === 'undefined') {
+      handleFetch = true;
+    }
+
```

Comment:
So the behavior here (reviewing the tests too) is `handleFetch` will always default to true unless the developer explicitly opts out. That's reasonable.